guides/v2.1/cloud/env/variables-intro.md

```diff
@@ -65,21 +65,23 @@ return array(
    ...
 );
 ```
-
 {% include note.html type="info" content="JS bundling and JS/CSS merging do not work with SCD on demand." %}
 
 ### `SKIP_HTML_MINIFICATION`
 
--  **Default**—`false`
+-  **Default**
+   - `true` for `ece-tools` 2002.0.13 and later
+   - `false` for earlier versions of `ece-tools`
 -  **Version**—Magento 2.1.4 and later
 
-Skip copying the static view files in the `var/view_preprocessed` directory to reduce downtime when deploying to the Staging and Production environments and generates minified HTML when requested.
+Determines whether to copy static view files to the `<magento_root>/init/` directory at the end of the build stage and whether HTML minification of static content occurs during deployment or on request. Set this value to `true` to reduce downtime when deploying to Staging and Production environments.
```

Comment:
`whether to copy static view files` -> `whether to copy minified HTML`
May be also good to add that this is not influencing performance, since this concern may arise.

Reply:
The SKIP_HTML_MINIFICATION variable is quite confusing because it joins the "copying files" with on-demand HTML minification --  The true and false explains how it works fairly clearly. 

 I think our docs could do a better job explaining this --maybe they do somewhere else, and we just need to provide a link here. 

Personally, I need to understand this process better, so I understand how things are related.